source/nodejs/adaptivecards-site/service-worker.js

```diff
@@ -0,0 +1,68 @@
+// Used pwa-builder as reference: https://github.com/pwa-builder/pwabuilder-serviceworkers/blob/master/serviceWorker5/pwabuilder-sw.js
+importScripts('/workbox/workbox-v6.5.3/workbox-sw.js');
```

Comment:
@paulcam206 By hosting these files on the website, we are hitting the `pr/pr#` url issue on the test site again. I don't think generating the service-worker dynamically is a good solution here, but we don't have access to the hexo url variable or `window` to grab the root of the url.

Do you have any ideas for resolving this?

Reply:
does `<%- config.root %>` not work here?

Reply:
alternatively, what happens if you refer to the path without the initial `/`?

Reply:
That worked, also fixed the manifest issue. Thanks!